apps/voting/app/src/components/VotingStatus.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import styled from 'styled-components'
+import { theme, IconTime, IconCross, IconCheck } from '@aragon/ui'
+
+const STATUSES = {
+  timeout: { label: 'Time out', Icon: IconTime, color: theme.textSecondary },
+  approved: { label: 'Approved', Icon: IconCheck, color: theme.positive },
+  rejected: { label: 'Rejected', Icon: IconCross, color: theme.negative },
+}
+
+const getStatus = (votesYes, votesNo) => {
+  if (votesYes === 0 && votesNo === 0) return STATUSES.timeout
```

Comment:
Wouldn't the timeout depend on the vote's end time rather than not getting an votes?